snuba/datasets/discover.py

```diff
@@ -85,22 +86,54 @@ def get_data_source(self):
         return None
 
     def get_columns(self) -> ColumnSet:
-        common = ColumnSet([])  # TODO: Fill out
+        common = ColumnSet([
+            ('event_id', FixedString(32)),
+            ('project_id', UInt(64)),
+            ('timestamp', DateTime()),
+            ('deleted', UInt(8)),
+            ('retention_days', UInt(16)),
+            ('platform', Nullable(String())),
+            ('sentry:release', Nullable(String())),
+            ('sentry:dist', Nullable(String())),
+            ('sentry:user', Nullable(String())),
+            # User
+            ('user_id', Nullable(String())),
+            ('username', Nullable(String())),
+            ('email', Nullable(String())),
+            ('ip_address', Nullable(String())),
```

Comment:
Does this coalesce the v4 and v6 columns for transactions?

Reply:
Good call, added this for both the transactions and Discover datasets